src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -107,7 +107,7 @@ public function getBaseUrl(string $path): string
      */
     protected function chooseBaseUrl(string $path): int
     {
-        return (int) fmod(hexdec(substr(hash('sha256', $path), 0, 10)), \count($this->baseUrls));
+        return crc32($path) % \count($this->baseUrls);
```

Comment:
I think we can change the distribution.
This implementation breaks on 32b archs, see appveyor (that's because crc32 returns negative numbers there, see doc)

Reply:
Would `abs(crc32($path)) % \count($this->baseUrls)` suffice?

Reply:
It should yes!